src/plugins/ui_feature/collaborative.ts

```diff
@@ -32,12 +40,20 @@ export class CollaborativePlugin extends UIPlugin {
     );
   }
 
-  getClient() {
-    return this.session.getClient();
+  getClient(clientId: ClientId) {
+    return this.session.getClient(clientId);
   }
 
-  getConnectedClients() {
-    return this.session.getConnectedClients();
+  getCurrentClient() {
+    return this.session.getCurrentClient();
+  }
+
+  getConnectedClients(): Set<Client> {
```

Comment:
I don't see any reason (anymore) for which it should be a Set rather than a simple array